### DISCUSSION TITLE
Problem with detection.

### DISCUSSION THREAD

**Discussion by Bongo2** [mcguirepr89/BirdNET-Pi#1179] [General]
I have been installing birdnet Pi on a Rasberry Pi 3B for a few days. As a source, I use the RTSP stream one of my surveillance camera of the REOLINK RLC-510A.
I like the program very much.

But sometimes the detection is not good. Actually, I only hear noise in these recordings:
![hc_2506](https://github.com/mcguirepr89/BirdNET-Pi/assets/45128074/8019b4b8-404a-4f43-ba13-83736d569ed0)
![hc_2505](https://github.com/mcguirepr89/BirdNET-Pi/assets/45128074/941d2974-0051-4c02-b6aa-e15e970e5b5c)
![hc_2504](https://github.com/mcguirepr89/BirdNET-Pi/assets/45128074/64aa96c1-573d-4ef6-8b8e-fe3737cdcb84)

Nevertheless, birds are detected to a high percentage. Birds that are so rare for me that it is actually not possible in my garden.
I have already tried both models. Of course, I could add the species to the "Excluded Species List" and increase the "minimum confidence". But it cannot be explained by these noise in the recording.
Can I adjust something?
Or is the microphone of my surveillance camera too bad?

**Comment by MueJosh** [comment]
The microphone is probably that bad... (looks really noisy)
you could change the location threshold 

But.. sometimes I have similar detections at night (with a dedicated soundcard and microphone)
All these "..dommel" birds show up at night - false positives are normal. 

You could try the new model (it is already implemented on [THIS](https://github.com/Nachtzuster/BirdNET-Pi) fork)

**Comment by Bongo2** [comment]
> you could change the location threshold

Where is this setting?

**Reply by MueJosh** [reply]
Tools - Settings 
Basic Settings - Model 

	Species Occurence Frequency Threshold [0.0005, 0.99]:

"More info" and the "species list tester" are really helpful! 


Screenshot with my settings (Nachtzuster fork) 
You definitely don't want to set it this low (if you just want your garden birds- set it to 0.01 or higher) 
![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/145150281/f2a50586-1bab-434d-879c-9892a4126b07)
